sormas-ui/src/main/java/de/symeda/sormas/ui/therapy/PrescriptionGrid.java

```diff
@@ -90,6 +90,9 @@ public Class<String> getType() {
 
 		getColumn(DOCUMENT_TREATMENT_BTN_ID).setHidden(isPseudonymized || !isEditAllowed);
 
+		if (!UserProvider.getCurrent().hasUserRight(UserRight.TREATMENT_CREATE)) {
```

Comment:
You can integrate this statement into the code in line 91, i.e. (isPseudonymized || !isEditAllowed || !UserProvider.getCurrent().hasUserRight(UserRight.TREATMENT_CREATE))

Reply:
Done